lti13migration/downloadcsv.php

```diff
@@ -0,0 +1,61 @@
+<?php
+// This file is part of the EQUELLA module - http://git.io/vUuof
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle. If not, see <http://www.gnu.org/licenses/>.
+
+require_once('../../../config.php');
+require_once($CFG->dirroot . '/mod/lti/classes/external.php');
+require_once($CFG->dirroot . '/lib/datalib.php');
+require_once($CFG->dirroot . '/mod/equella/lib.php');
+
+global $DB;
+
+header('Content-Type: text/csv; charset=utf-8');
+header('Content-Disposition: attachment; filename=courses.csv');
+
+// Cannot use `$DB0->get_field` due to issue <https://tracker.moodle.org/browse/MDL-27629>.
+$oeqMoodleModuleID = $DB->get_field_sql("SELECT id FROM {modules} WHERE name = 'equella'");
```

Comment:
What's this curly brace syntax (e.g. `FROM {modules} WHERE`)? Is it some kind of PHP DB abstraction layer?

Reply:
This is a Moodle specific syntax. Moodle will convert `{modules}` to `mdl_modules` where `mdl` is the table prefix.